components/CardList/CardList.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { slugify } from 'utils';
+import { Box, CardGrid, Card, Loader } from 'ui-kit';
+import { CustomLink } from 'components';
+
+const CardwithBackground = (props = {}) => (
+  <Card
+    as="a"
+    onClick={props.handleOnClick}
+    display="flex"
+    alignItems={'center'}
+    justifyContent={'center'}
+    p={'l'}
+    flexDirection={'column'}
+    backgroundImage={`linear-gradient(rgba(0, 0, 0, 0.33), rgba(71, 71, 71, 0.85)),url(${props.coverImage})`}
+    backgroundSize={'cover'}
+    color={'white'}
+    maxWidth={'300px'}
+  >
+    <Box as={'h1'} display={'flex'}>
+      {props.title}
+    </Box>
+    <Box as={'p'} display={'flex'}>
+      {props.summary}
+    </Box>
+  </Card>
```

Comment:
You don't need to do the brackets around the string. You can just do the string.

`alignItems="center"` instead of `alignItems={'center'}`.